db/migrate/20151123235600_add_purchase_type_to_gsa18f_procurements.rb

```diff
@@ -0,0 +1,11 @@
+class AddPurchaseTypeToGsa18fProcurements < ActiveRecord::Migration
+  def change
+    add_column :gsa18f_procurements, :purchase_type, :integer
+
+    execute <<-SQL
+      UPDATE gsa18f_procurements SET purchase_type = 0 WHERE purchase_type IS NULL;
+      SQL
```

Comment:
I thought it would, too! but when I ran this migration locally it did not set the default for existing records.